src/components/v2/Table/index.tsx

```diff
@@ -0,0 +1,72 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
+import TableMUI from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import Head from './Head';
+import { useStyles } from './styles';
+
+interface ITableRowProps {
+  key: string;
+  render: () => React.ReactNode | string;
+  value: string | number | boolean;
+}
+
+interface ITableProps {
+  title: string;
+  data: ITableRowProps[][];
+  columns: { id: string; label: string; orderable: boolean }[];
+  fetchData?: (params: { order: { by: string; direction: 'asc' | 'desc' } }) => void;
+}
+
+export function Table({ columns, data, fetchData, title }: ITableProps) {
+  const styles = useStyles();
+  const [orderBy, setOrderBy] = useState<typeof columns[number]['id'] | undefined>(undefined);
+  const [order, setOrder] = useState<'asc' | 'desc' | undefined>('asc');
```

Comment:
A small thing, but how about `orderDirection` to keep the same nomenclature you used for `orderBy`?